spec/features/reviews/user_can_delete_reviews_from_shelter_show_page_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe "reviews delete process" do
+  it "I see a link on shelters show page that can to delete review" do
+
+      visit("/shelters/#{@shelter_2.id}")
+
```

Comment:
I think if we use parentheses here then we need to make it uniform across all tests